scripts/train_lora.sh

```diff
@@ -0,0 +1,30 @@
+deepspeed --num_gpus 6 train_lora.py \
```

Comment:
Update the relative path? or you can just add this to readme and delete this script.

Other things look good to me.